.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -35,4 +40,7 @@ or AWS CLI commands to trigger something.
 - [ ] Fix linting errors
 - [ ] Make sure code coverage hasn't dropped
```

Comment:
Imho yes because I want people to think about this without having to go through travis or Coveralls. It should be on their mind as early as possible so they don't even have to go through the tooling to tell them it failed.